tools/functional-tester/etcd-tester/tester.go

```diff
@@ -83,6 +83,16 @@ func (tt *tester) runLoop() {
 			plog.Printf("[round#%d case#%d] recovered failure", i, j)
 
 			if !tt.consistencyCheck {
+				plog.Printf("[round#%d case#%d] canceling the stressers...", i, j)
```

Comment:
@xiang90 because 

```
2016-04-26 21:52:23.163016 I | etcd-tester: [round#1090 case#4] start failure kill one member for long time and expect it to recover from incoming snapshot
2016-04-26 21:52:23.163046 I | etcd-tester: [round#1090 case#4] start injecting failure...
2016-04-26 21:52:23.177334 I | etcd-tester: [Stress error] rpc error: code = 13 desc = etcdserver: server stopped (10.240.0.14:2379)
2016-04-26 21:52:23.177379 I | etcd-tester: [Stress error] rpc error: code = 13 desc = etcdserver: server stopped (10.240.0.14:2379)
2016-04-26 21:52:23.177397 I | etcd-tester: [Stress error] rpc error: code = 13 desc = etcdserver: server stopped (10.240.0.14:2379)
2016-04-26 21:52:23.177415 I | etcd-tester: [Stress error] rpc error: code = 13 desc = etcdserver: server stopped (10.240.0.14:2379)
2016-04-26 21:52:23.177441 I | etcd-tester: [Stress error] rpc error: code = 13 desc = etcdserver: server stopped (10.240.0.14:2379)
2016-04-26 21:52:23.177461 I | etcd-tester: [Stress error] rpc error: code = 13 desc = etcdserver: server stopped (10.240.0.14:2379)
2016-04-26 21:52:23.177482 I | etcd-tester: [Stress error] rpc error: code = 13 desc = etcdserver: server stopped (10.240.0.14:2379)
2016-04-26 21:52:25.821911 I | etcd-tester: [Stress error] rpc error: code = 13 desc = transport is closing (10.240.0.14:2379)
2016-04-26 21:52:25.821951 I | etcd-tester: [Stress error] rpc error: code = 13 desc = transport is closing (10.240.0.14:2379)
2016-04-26 21:52:25.821966 I | etcd-tester: [Stress error] rpc error: code = 13 desc = transport is closing (10.240.0.14:2379)
2016-04-26 21:52:25.821978 I | etcd-tester: [Stress error] rpc error: code = 13 desc = transport is closing (10.240.0.14:2379)
2016-04-26 21:52:25.821990 I | etcd-tester: [Stress error] rpc error: code = 13 desc = transport is closing (10.240.0.14:2379)
2016-04-26 21:52:25.821997 I | etcd-tester: [Stress error] rpc error: code = 13 desc = transport is closing (10.240.0.14:2379)
```

When we inject failures, stressers return with such errors, and if we don't restart the stressors, we slowly run out of stressors and get `too slow no commit` error. That is why I think we need to restart stressers for every case when we run testing cluster without consistency check.